week-1/day-1/boolean_to_string.rb

```diff
@@ -0,0 +1,6 @@
+def bool_to_word bool
+    if bool
+        return "Yes"
+    end
+    return "No"
```

Comment:
Esta función está bien, pero hay unas cuantas maneras de mejorarla.

La primera es incluir el `return "No"` dentro de la misma función con un `else`. Los métodos en Ruby no necesitan de `return` para devolver un valor. Tu función podria haber quedado simplemente así:

```
if bool
  "Yes"
else
  "No"
end
```

La segunda es más corta y un poco más avanzada. Hace uso del [operador ternario](https://www.rubyguides.com/2019/10/ruby-ternary-operator/). Es como un `if else` pero resumido y solo funciona si tienes dos posibles resultados en la función.

Se declara así: `condition ? true : false`

Si puedes, revisa el link para que veas mejor como funciona y como podrías escribir la funcion usándola.

Reply:
En cuanto al primer punto que tocas, te parece inadeacuado en cuestiòn de estructura omitir la sentencia else?(en este caso en partciular podrìa quedar asì).
```suggestion
    if bool
        return "Yes"
    end
    "No"
```
En cuanto a los operadores ternarios, por un tema visual y de orden (para mi) no los uso, reocmiendas usarlos estrictamente cuando se presenten estos casos y evitar conditional statements como el que propuse anteriormente?

Reply:
No estaba seguro que supieras sobre las otras maneras de hacerlo por eso las sugerí. La forma en la que lo resolviste inicialmente parece bastante código JS. No hay nada de malo con eso.

Tienes razón en que es una cuestión de gustos. Personalmente, yo prefiero usar el operador ternario cuando tengo la oportunidad porque lo veo muy limpio y rápido.

Creo que hasta que no nos recomienden usar una guía de estilos vamos a seguir teniendo conversaciones como estas seguido porque hay varias formas de llegar a lo mismo y va a seguir siendo cuestión de preferencias.

Reply:
Tienes toda la razòn - de todas formas apreciarìa mucho poder compartir  disitintas perspectivas. De hecho creo que es hora de empezar a usar esos operadores jaja. Gracias por los comentarios.

PD : en la documentaciòn de Ruby hay un libro sugerido - hacen referencia sobre estàndares y convenciones dentro del lenguaje, serìa bueno poder revisarlo (http://docs.ruby-doc.com/docs/ProgrammingRuby/)